src/controllers/__tests__/organizationController.test.ts

```diff
@@ -222,36 +278,42 @@ describe("OrganizationController", () => {
       req.user = mockedLoggedUser;
       req.body = mockedOrganizationRegistrationParams;
       const organizationController = await getOrganizationController();
-      const result = await organizationController.registerOrganization(req);
-      expect(result).toEqual({
+      const result = await organizationController
+        .registerOrganization(req)
+        .run();
+      expect(isLeft(result)).toBeTruthy();
+      expect(result.value).toEqual({
         apply: expect.any(Function),
         detail: expect.any(String),
         kind: "IResponseErrorForbiddenNotAuthorized"
       });
     });
 
-    it("should return a validation error if the organization parameters are invalid", async () => {
+    it("should return a left task with a validation error response if the organization parameters are invalid", async () => {
```

Comment:
I'd prefer the previous description text (since it left implmentation details out of the test description)